sparkler-api/src/main/java/edu/usc/irds/sparkler/util/FetcherDefault.java

```diff
@@ -39,6 +68,15 @@
     public FetchedData fetch(Resource resource) throws Exception {
         LOG.info("DEFAULT FETCHER {}", resource.getUrl());
         URLConnection urlConn = new URL(resource.getUrl()).openConnection();
+        if (httpHeaders != null){
+            httpHeaders.entrySet().forEach(e -> urlConn.setRequestProperty(e.getKey(), e.getValue()));
+        }
+        String userAgentValue = getUserAgent();
+        if (getUserAgent() != null) {
+            LOG.debug(USER_AGENT + ": " + userAgentValue);
+            urlConn.setRequestProperty(USER_AGENT, userAgentValue);
```

Comment:
So, if we go by this, we are ignoring the default `User-Agent` set in the `fetcher.headers`. I think we should add the default one in the `fetcher.user.agents` list (set is better) while loading the configuration. What do you think?